### PR TITLE
feat: Bump logging-operator and fluent-bit

### PR DESCRIPTION
**Pull request by mesosphere-teamcity**
This upgrades logging-operator to 3.17.9 to use the latest upstream chart logging-operator 3.17.9.

### CODE REVIEW

File: services/logging-operator/3.17.9/logging-operator.yaml [hunk 1]
```diff
@@ -13,7 +12,7 @@ spec:
         name: kubernetes-charts.banzaicloud.com
         namespace: kommander-flux
       # When upgrading, make sure to update the fluent-bit and fluentd images in logging-operator-logging.yaml.
```
Comment (bcmendoza) [inline services/logging-operator/3.17.9/logging-operator.yaml:14]: TODO

**Review by bcmendoza — CHANGES_REQUESTED** [review]

File: services/logging-operator/3.17.9/logging-operator.yaml (hunk 1 above)
Comment (gracedo) [reply services/logging-operator/3.17.9/logging-operator.yaml:14]: done in https://github.com/mesosphere/kommander-applications/pull/651/commits/4a9679559314f388a0b5b1b30b98ba78d4fc3eff

File: services/fluent-bit/0.20.9/resource-quota.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ spec:
   wait: true
   interval: 6h
   retryInterval: 1m
-  path: ./services/fluent-bit/0.20.5/resource-quota
+  path: ./services/fluent-bit/0.20.9/resource-quota
```
Comment (gracedo) [inline services/fluent-bit/0.20.9/resource-quota.yaml:12]: @bcmendoza fyi (same deal as my comment on the other pr) this is another one that I manually updated

**Review by gracedo — APPROVED** [review]

**Comment by gracedo** [comment]
https://github.com/mesosphere/kommander/pull/2417 is ✅

**Review by bcmendoza — APPROVED** [review]

**Pull request merged**